### ISSUE TITLE
month previous / next buttons don't work

### ISSUE THREAD

**Issue by JernejL** [joshsalverda/datepickr#10]
All the buttons do is hide the calendar instead of changing month, must been broken in some recent release.

**Comment by joshsalverda** [comment]
Yep the latest code in the repo is unfortunately broken (my bad), but not the latest release. You can use the latest release version from here: https://github.com/joshsalverda/datepickr/releases

**Comment by burlesona** [comment]
:+1: experienced the same thing. Thanks for the link to the correct code, you might want to put that at the top of the Readme or something :)

**Comment by burlesona** [comment]
Actually I'm still seeing the same issue on the linked release. The next/previous month buttons do change the month but they also hide the calendar.

**Comment by deafwood** [comment]
I had the same problem so I copied the .js from the demo - you can get it here: http://joshsalverda.github.io/datepickr/src/datepickr.min.js

I'm not sure what version this one is - but it works for me.
